modules/sbo-connecting-spring-petclinic-sample-app-to-postgresql-database-service.adoc

```diff
@@ -42,81 +37,50 @@ The output verifies that the `ServiceBinding` CR is created to project the bindi
-    },
-    {
-      "lastTransitionTime": "2021-09-06T13:42:28Z",
-      "message": "",
-      "reason": "ApplicationsBound",
-      "status": "True",
-      "type": "Ready",
-    },
-  ],
-]
+NAME                          READY   REASON              AGE
+spring-petclinic-postgresql   True    ApplicationsBound   47m
 ----
 +
-By default, the values from the binding data of the database service are projected as files into the workload container that runs the sample application.
+By default, the values from the binding data of the database service are projected as files into the workload container that runs the sample application. For example, all the values from the Secret resource are projected into the `bindings/spring-petclinic-pgcluster` directory.
 
-. To verify that the files in the application contain the projected binding data, use the following command in shell to print out the contents:
+. Verify that the files in the application contain the projected binding data, by using the following command in shell to print out the contents:
 +
 ----
-$ for i in username password host port type; do oc exec -it deploy/spring-petclinic-rest -n my-postgresql -- /bin/bash -c 'find /bindings/*/'$i' -exec echo -n {}:" " \; -exec cat {} \;'; echo; done
+$ for i in username password host port type; do oc exec -it deploy/spring-petclinic -n my-petclinic -- /bin/bash -c 'find /bindings/*/'$i' -exec echo -n {}:" " \; -exec cat {} \;'; echo; done
 ----
 +
 .Example output: With all the values from the secret resource
 ----
-/bindings/spring-petclinic-rest/username: hippo
-/bindings/spring-petclinic-rest/password: w0ZB<0j1W|K;+4*TlK7-w^z/
-/bindings/spring-petclinic-rest/host: hippo-pgbouncer
-/bindings/spring-petclinic-rest/port: 5432
-/bindings/spring-petclinic-rest/type: postgresql
+/bindings/spring-petclinic/username: hippo
+/bindings/spring-petclinic/password: w0ZB<0j1W|K;+4*TlK7-w^z/
+/bindings/spring-petclinic/host: hippo-pgbouncer
+/bindings/spring-petclinic/port: 5432
+/bindings/spring-petclinic/type: postgresql
 ----
 
 . Set up the port forwarding from the application port to access the sample application from your local environment:
 +
 ----
-$ oc port-forward --address 0.0.0.0 svc/spring-petclinic-rest 9966:80 -n my-postgresql
-----
-
-. Access link:http://localhost:9966/petclinic[http://localhost:9966/petclinic].
-+
-You can now remotely access the Spring PetClinic sample application at localhost:9966.
-
-. To verify that the application is now connected to the database service, access the list of all pets:
-+
-----
-$ curl -X GET "http://localhost:9966/petclinic/api/pets" -H "accept: application/json"
+$ oc port-forward --address 0.0.0.0 svc/spring-petclinic 8080:80 -n my-petclinic
 ----
 +
 .Example output
 ----
-[{"id":1,"name":"Leo","birthDate":"2000/09/07","type":{"id":1,"name":"cat"},
-"owner":{"id":1,"firstName":"George","lastName":"Franklin","address":"110...
+Forwarding from 0.0.0.0:8080 -> 8080
+Handling connection for 8080
 ----
+
```

Comment:
Would it make sense to combine the following lines into a single step and put them in a new `.Verification` section?